src/feature_annotation/score_genes_cell_cycle_scanpy/script.py

```diff
@@ -0,0 +1,28 @@
+import scanpy as sc
+import mudata as mu
+
+
+mudata = mu.read(f'{par["input"]}')
+
+par["use_raw"] = par.setdefault("use_raw", None)
+par["gene_pool"] = par.setdefault("gene_pool", None)
+
+s_genes = [x.strip() for x in open(par["s_genes"])]
```

Comment:
Could you use a context manager for this? `with open(par["s_genes"]) as open_s_genes_file`. This makes sure that the file is properly closed if something goes wrong.